test/memote.jl

```diff
@@ -70,3 +70,22 @@ end
 
     @test length(reactions_transport_no_gpr(model; config = memote_config)) == 4
 end
+
+@testset "Biomass" begin
+    @test model_has_atpm_reaction(model)
+    wrong_model = convert(StandardModel, model)
+    remove_reaction!(wrong_model, "ATPM")
+    @test !model_has_atpm_reaction(wrong_model)
+
+    @test all(model_has_growth_atp_in_biomass(model))
```

Comment:
looking at it, would it be more useful to return dictionaries out of the tests? as in
```
Dict("biomass1" => true, "weirdmass" => false)
```

Reply:
aahh, yeah fixinig